src/components/portal/portal.tsx

```diff
@@ -78,6 +79,18 @@ export class Portal {
     @Prop()
     public inheritParentWidth = false;
 
+    /**
+     * Set `showArrow` to `true` if you need to show sourse of `limel-popover` or `limel-tooltip`
```

Comment:
```suggestion
     * Set `showArrow` to `true` if you need to show source of `limel-popover` or `limel-tooltip`
```

I think this description can be improved a bit apart from the typo. But I'm a little to tired to come up with something better myself right now. Maybe tomorrow… 😅

Reply:
⚡